modules/special-rules/pages/assault-1.adoc

```diff
@@ -0,0 +1,9 @@
+= Assault +1
+
+These units are particularly fierce melee fighters, without sacrificing their ranged capability -- such as Space Wolves Long Fangs or Grey Hunters.
+
+---
+
+Apply this rule to units that have the {assault-1} ability, relative to their base units:
+
+* Add +1 to their Assault value.
```

Comment:
```suggestion
* Add +1 to their Assault value.

NOTE: In 3rd edition this ability was called Space Wolves.
```

Reply:
We should merge this with the other idea above, of course. I suspect it is probably best to put the note either just before or after the horizontal rule that separates the intro/summary/fluff from the actual rules content.

Reply:
Suggestion added above, marking this as resolved.